src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/client/payhub/PayHubClient.java

```diff
@@ -17,6 +18,7 @@
     url = "${pay-hub.api.url}",
     configuration = PayHubClientConfiguration.class
 )
+@Profile({"integration", "!nosb"})
```

Comment:
This is application code - it should have `integration` profile. Same for all other classes from `src/main`

Reply:
It is called in integration tests and otherwise integration tests fail